syslog/modules/lpeg/syslog_message.lua

```diff
@@ -732,13 +732,13 @@ prog_grammar["sudo"] = l.Ct(
 prog_grammar["systemd-logind"] = l.Ct(
                                (
                                  l.P"New session "
-                               * l.Cg(integer, "session_id")
+                               * l.P"c"^-1 * l.Cg(integer, "session_id")
                                * l.P" of user "
                                * capture_followed_by("user_id", "." * l.P(-1))
                                * l.Cg(l.Cc("SESSION_START"), "sd_message")
                                ) + (
                                  l.P"Removed session "
-                               * l.Cg(integer, "session_id")
+                               * l.P"c"^-1 * l.Cg(integer, "session_id")
```

Comment:
I realize this file has a bunch of repetition but lets start improving the situation by making this a single session_id grammar used in both places, thanks.